{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -62,6 +62,8 @@
 # https://docs.djangoproject.com/en/dev/ref/middleware/#x-content-type-options-nosniff
 SECURE_CONTENT_TYPE_NOSNIFF = env.bool("DJANGO_SECURE_CONTENT_TYPE_NOSNIFF", default=True)
 
+STORAGES = dict()
```

Comment:
I think this overrides the [default value](https://docs.djangoproject.com/en/4.2/ref/settings/#storages). If the user selects `cloud_provider='None'`, none of the branches below are hit, and the setting will probably be invalid.